app/src/main/java/com/hedvig/app/feature/birthday/viewmodel/BirthdayViewModel.kt

```diff
@@ -0,0 +1,50 @@
+package com.hedvig.app.feature.birthday.viewmodel
+
+import androidx.lifecycle.MutableLiveData
+import androidx.lifecycle.ViewModel
+import com.hedvig.app.feature.birthday.data.BirthdayRepository
+import io.reactivex.disposables.CompositeDisposable
+import io.reactivex.rxkotlin.plusAssign
+import timber.log.Timber
+import java.util.*
+
+
+class BirthdayViewModel(
+    private val birthdayRepository: BirthdayRepository
+) : ViewModel() {
+
+    private lateinit var birthDate: String
+
+    val isBirthdayData: MutableLiveData<Boolean> = MutableLiveData()
+    private val disposables = CompositeDisposable()
+
+
+
+    private fun isBirthday(): Boolean {
+
+        val birthMonth = birthDate.substring(4, birthDate.length - 6).toInt()
```

Comment:
If we make this function a bit more pure in line with my suggestion around the `birthDate`-parameter, we could move it to a companion object or to its own file, allowing it to be reused a little bit better. 🙂